backend/src/gpml/auth0_util.clj

```diff
@@ -31,28 +72,34 @@
         j/read-value
         (get "access_token"))))
 
-(defn maybe-refresh-token-fetch-auth0-users [auth0-config]
+(defn maybe-refresh-token-fetch-auth0-users [auth0-config fetch-auth0-users-opts]
```

Comment:
another feeling here is that we currently don't have a really good design or good (var and function) naming in this namespace and maybe we are keeping the tendency ... `(fetch-auth0-users domain fetch-auth0-users-opts)` 😬

Reply:
Yes, I also feel that this namespace needs a bit of refactoring, but that is out of the scope of this issue, I guess, and that is why I kept the tendency for now.